src/Context/ZendObserverFiber.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/** @noinspection PhpUndefinedClassInspection */
+/** @noinspection PhpUndefinedNamespaceInspection */
+/** @phan-file-suppress PhanUndeclaredClassReference */
+/** @phan-file-suppress PhanUndeclaredClassCatch */
+/** @phan-file-suppress PhanUndeclaredClassMethod */
+/** @phan-file-suppress PhanUndeclaredMethod */
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Context;
+
+use FFI;
+use FFI\Exception;
+use OpenTelemetry\SDK\Behavior\LogsMessagesTrait;
+use OpenTelemetry\SDK\EnvironmentVariablesTrait;
+
+class ZendObserverFiber
+{
+    use LogsMessagesTrait;
+    use EnvironmentVariablesTrait;
+
+    protected static $fibers = null;
+
+    public function isEnabled(): bool
+    {
+        return (
+            PHP_VERSION_ID >= 80100 &&
+            $this->getBooleanFromEnvironment('OTEL_PHP_FIBERS_ENABLED', false) &&
+            class_exists(FFI::class)
+        );
+    }
+
+    /**
+     * @psalm-suppress UndefinedClass
+     */
+    public function init(): bool
+    {
+        if (null === self::$fibers) {
+            try {
+                $fibers = FFI::load(__DIR__ . '/fiber/zend_observer_fiber.h');
```

Comment:
Removing `FFI::scope()` breaks this for php-fpm etc. with the default FFI configuration ([ffi.enable=preload](https://www.php.net/manual/en/ffi.configuration.php#ini.ffi.enable)).

Reply:
ok, put this back to how it was try(scope)/try(load).